src/core/Util.js

```diff
@@ -130,6 +130,24 @@ module.exports = {
     var wysiwygDivObserver = new MutationObserver(handleChange);
 
     wysiwygDivObserver.observe(wysiwygDiv, observerConfig);
+  },
+
+  hideFooterWhenTypingOnMobile: function() {
+    var inputArea = $('input, textarea, .wk-wysiwyg');
+
+    if(window.innerWidth <= 992) {
+      inputArea.addClass('input-area');
+
+      var _input = $('.input-area');
```

Comment:
Unnecessary operation. Instead of adding class to `inputArea` and then querying elements that match that class creates redundant overhead. `_input` -> `inputArea`